core/sparse_array.cpp

```diff
@@ -95,6 +140,18 @@ void SparseArray::Reset(unsigned int p_nrows, unsigned int p_ncols)
 	finished_ = false;
 }
 
+void SparseArray::Resize(uint32_t p_row)
+{
+    //vectors are reallocated by 1000 entries at a time. It can be wasteful but probably better than making a function call for each new entry. Will defer the final decision to Ben. 
```

Comment:
This comment says "by 1000 entries at a time", but the code doubles the capacity with each realloc.  Doubling is probably reasonable; you know you're not wasting more than 50% of memory in the worst case, and it avoids a large number of reallocs if the final size is >> initial_width.  Most STL implementations of things like std::vector double the capacity when it is exceeded, as I understand it.

Reply:
The comment here remains inaccurate, as discussed above; please fix it to reflect the behavior of the code.